test/xpu_api/numerics/complex.number/complex.ops/complex_divide_complex.pass.cpp

```diff
@@ -69,15 +69,17 @@ void test_edges()
                 switch (classify(testcases[j]))
                 {
                 case zero:
+#if !_PSTL_TEST_COMPLEX_DIV_COMPLEX_BROKEN && !_PSTL_TEST_COMPLEX_DIV_COMPLEX_BROKEN_GLIBCXX
```

Comment:
I see here you've added both `!_PSTL_TEST_COMPLEX_DIV_COMPLEX_BROKEN && !_PSTL_TEST_COMPLEX_DIV_COMPLEX_BROKEN_GLIBCXX`.  Is that intentional?

This is the only place `!_PSTL_TEST_COMPLEX_DIV_COMPLEX_BROKEN` is added to an assert within this PR

Reply:
Fixed